src/main/java/com/axonivy/connector/aws/authentication/Aws4AuthenticationRequestFilter.java

```diff
@@ -33,7 +33,7 @@ public void filter(ClientRequestContext context) throws IOException {
       headers.add(X_AMZ_DATE, signer.getTimeStamp());
       headers.add(AUTHORIZATION, signer.sign());
     } catch (NoSuchAlgorithmException | InvalidKeyException ex) {
-      throw new IOException("Could not sign Amazon AWS request", ex);

```

Comment:
since this feature is not only use to talk with Amazon AWS, also with services like minio (which is implementing s3 interface), I removed this term here